### ISSUE TITLE
Add an Add button or forms for all database models

### ISSUE THREAD

**Issue by postmodern** [ronin-rb/ronin-app#93]
Add an Add button/link or a form to all database views. This form could possibly be added to the top of all `db/*/index.erb` views. The add form could also have it's own page which is linked to from their respective `index.erb` pages.

**Comment by AI-Mozi** [comment]
Is this simple form enough? Or maybe you want it to look different?

<img width="1394" alt="Screenshot 2024-04-03 at 16 30 16" src="https://github.com/ronin-rb/ronin-app/assets/47834533/bb98a3cb-85a4-4a3a-8606-8fbf5bc0ebfa">

**Comment by postmodern** [comment]
@AI-Mozi yes, perfect! I would move the `Import` button so it's on the same line as the input field. You can use a `div` and Bulma's `field has-addons` classes (see `views/spider.erb` for an example).

**Comment by postmodern** [comment]
Also, here are the database models that implement a `self.import` method:

```
lib/ronin/db/address.rb:      def self.import(address)
lib/ronin/db/advisory.rb:      def self.import(id)
lib/ronin/db/cert.rb:      def self.import(cert)
lib/ronin/db/cert_issuer.rb:      def self.import(name)
lib/ronin/db/cert_name.rb:      def self.import(name)
lib/ronin/db/cert_subject.rb:      def self.import(name)
lib/ronin/db/credential.rb:      def self.import(cred)
lib/ronin/db/email_address.rb:      def self.import(email)
lib/ronin/db/host_name.rb:      def self.import(name)
lib/ronin/db/organization.rb:      def self.import(name)
lib/ronin/db/password.rb:      def self.import(password)
lib/ronin/db/person.rb:      def self.import(name)
lib/ronin/db/phone_number.rb:      def self.import(number)
lib/ronin/db/port.rb:      def self.import(number)
lib/ronin/db/service.rb:      def self.import(name)
lib/ronin/db/url.rb:      def self.import(uri)
lib/ronin/db/user_name.rb:      def self.import(name)
spec/model/importable_spec.rb:    def self.import(value)
```

**Comment by AI-Mozi** [comment]
@postmodern thanks for the list

**Comment by postmodern** [comment]
Implemented by PR #98.
